lib/index.js

```diff
@@ -111,28 +112,13 @@ function getPolyfillString(options) {
 	const ua = new UA(options.uaString);
 	const uaDebugName = ua.getFamily() + '/' + ua.getVersion() + ((ua.isUnknown() || !ua.meetsBaseline()) ? ' (unknown/unsupported; using policy `unknown='+options.unknown+'`)' : '');
 	const lf = options.minify ? '' : '\n';
-	let explainerComment = [];
+	const allWarnText = 'Using the `all` alias with polyfill.io is a very bad idea. In a future version of the service, `all` will deliver the same behaviour as `default`, so we recommend using default instead.';
+	const outputString = options.stream ? new ReadableStream : [];
```

Comment:
When creating a ReadableStream you need to implement the `_read` method yourself. I believe this is why you are getting the exception `Error: not implemented`. The NodeJS docs explain it better than I can -- https://nodejs.org/api/stream.html#stream_implementing_a_readable_stream